src/components/ImportJobs/components/FileUpload/FileUpload.js

```diff
@@ -1,35 +1,125 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import noop from 'lodash/noop';
-
+import ReactDropzone from 'react-dropzone';
 import { Button } from '@folio/stripes/components';
 
+import classNames from 'classnames/bind';
 import css from './FileUpload.css';
 
-class FileUpload extends React.Component {
-  static propTypes = {
-    title: PropTypes.string.isRequired,
-    uploadBtnText: PropTypes.string.isRequired,
-  };
-
-  render() {
-    const {
-      title,
-      uploadBtnText,
-    } = this.props;
-
-    return (
-      <div className={css.upload}>
-        <span className={css.uploadTitle}>{title}</span>
-        <Button
-          buttonStyle="primary"
-          onClick={noop}
-        >
-          {uploadBtnText}
-        </Button>
-      </div>
-    );
-  }
-}
+
+const cx = classNames.bind(css);
+
+const getTitleClassName = dropZoneState => {
+  return cx({
+    uploadTitle: true,
+    activeUploadTitle: dropZoneState
+  });
+};
+
+const getUsedStyle = (styleFromProps, classNameFromProps, initialStyle) => {
+  if (styleFromProps) return styleFromProps;
+  if (classNameFromProps) return null;
+
+  return initialStyle;
+};
+
+const initStyle = {
```

Comment:
Can we have this as initClass? I just do not want to have styles block defined not inside `css` file.